### ISSUE TITLE
Fresh install Julia 1.1: test errors -> No method iterate(::Chain), etc.

### ISSUE THREAD

**Issue by McCoyBecker** [TuringLang/Turing.jl#713]
Plenty of errors throwed.

Examples:

1. No method iterate(::Chain...)
2. LoadError: ArgumentError: at least one finite value must be provided to formatter.

Also mentions that it can't find Turing on the current path, which is bizarre because I just added it in pkg.

Is this a Julia 1.1 thing?

I should motivate this post by saying that I completely wiped Julia because I couldn't get the Visualize Samplers code to work on the website, because of no method iterate errors.

Any help would be greatly obliged.

Here's more information on the 1. No method iterate issue from test.

beta_binomial.jl: Error During Test at /home/mccoy/.julia/packages/Turing/dsVcO/test/utility.jl:105
  Got exception outside of a @test
  LoadError: MethodError: no method matching iterate(::Chains{Union{Missing, Float64},Float64,NamedTuple{(:parameters,),Tuple{Array{String,1}}},NamedTuple{(:hashedsummary,),Tuple{Base.RefValue{Tuple{UInt64,MCMCChains.ChainSummaries}}}}})
  Closest candidates are:
    iterate(!Matched::Core.SimpleVector) at essentials.jl:568
    iterate(!Matched::Core.SimpleVector, !Matched::Any) at essentials.jl:568
    iterate(!Matched::ExponentialBackOff) at error.jl:199

**Comment by cpfiffer** [comment]
We changed some indexing behavior over at [MCMCChains](https://github.com/TuringLang/MCMCChains.jl), specifically in [PR 57](https://github.com/TuringLang/MCMCChains.jl/pull/57). It's not a Julia 1.1 issue, it's an oversight on my part to adapt the tests.

I've got a fix ready to go after testing completes, but for the most part, Turing should work fine. The `plot_samplers` function in the visualization library is also out of date for the indexing functionality, and I'll follow up shortly with an updated version of that function that should work for you.

**Comment by McCoyBecker** [comment]
Awesome! Thank you.

**Comment by McCoyBecker** [comment]
Apologies for closing, will let you close when good.

**Comment by cpfiffer** [comment]
As to the sampler visualization, try changing `plot_sampler` to 

```julia
function plot_sampler(chain)
    # Extract values from chain.
    val = get(chain, [:s, :m, :lp])
    ss = link.(Ref(InverseGamma(2,3)), val.s)
    ms = val.m
    lps = val.lp

    # How many surface points to sample.
    granularity = 500

    # Range start/stop points.
    spread = 0.5
    σ_start = minimum(ss) - spread * std(ss); σ_stop = maximum(ss) + spread * std(ss);
    μ_start = minimum(ms) - spread * std(ms); μ_stop = maximum(ms) + spread * std(ms);
    σ_rng = collect(range(σ_start, stop=σ_stop, length=granularity))
    μ_rng = collect(range(μ_start, stop=μ_stop, length=granularity))

    # Make surface plot.
    p = surface(σ_rng, μ_rng, evaluate,
          camera=(30, 65),
          ticks=nothing,
          colorbar=false,
          color=:inferno)

    line_range = 1:length(ms)

    plot3d!(ss[line_range], ms[line_range], -lps[line_range],
        lc =:viridis, line_z=collect(line_range),
        legend=false, colorbar=false, alpha=0.5)

    return p
end
```

I'll update it on the website shortly.

**Comment by cpfiffer** [comment]
This is solved on master after #715. Thanks for bringing this up!
